Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -133,6 +133,7 @@ class TabLocationView: UIView {
         readerModeButton.addGestureRecognizer(UILongPressGestureRecognizer(target: self, action: #selector(TabLocationView.SELlongPressReaderModeButton(_:))))
         readerModeButton.isAccessibilityElement = true
         readerModeButton.accessibilityLabel = NSLocalizedString("Reader View", comment: "Accessibility label for the Reader View button")
+        readerModeButton.accessibilityIdentifier = "TabLocationView.readerModeButton"
```

Comment:
I don't think running UI tests in other languages is one of our goals; these tests are for UI correctness, not string correctness (we have separate screenshot tests with localizer review for the latter). But even if we do decide to run tests in other languages, I don't think identifiers are the right solution. See [this thread](https://github.com/kif-framework/KIF/issues/243) for more details.